Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
I'm happy to try it without the `-R` but I fear we'll end up putting it in at some point. There's a lot of potential trip-points on a directory sharing non-root & root stuff.